pkg/kubectl/describe.go

```diff
@@ -927,6 +935,9 @@ func describeContainers(label string, containers []api.Container, containerStatu
 				}
 				fmt.Fprintf(out, "      %s:\t%s (%s:%s)\n", e.Name, valueFrom, e.ValueFrom.FieldRef.APIVersion, e.ValueFrom.FieldRef.FieldPath)
 			case e.ValueFrom.ResourceFieldRef != nil:
+				if nodeCapacity != nil {
+					fieldpath.MergeContainerResourceLimitsWithCapacity(&container, nodeCapacity)
```

Comment:
Do I understand this correctly that for a 4 GB node I will see a 4GB limit in the pod output as limit? Wouldn't it make sense to print that more explicitly? Something like:

Memory limits: unlimited, up to node capacity 4GB